templates/site/js/dataproduct-form.js

```diff
@@ -905,7 +905,9 @@ function calendarDatesEqual(checkboxDateEqualID, startDateID, endDateID){
         endDateElement.setAttribute("disabled", "disabled");
 
         if(startDateElement.value != ""){
-            endDateElement.value = startDateElement.value;
+            var firstDate = new Date(startDateElement._flatpickr.selectedDates[0])
+            endDateElement._flatpickr.selectedDates[0] = startDateElement._flatpickr.setDate(firstDate);
+            endDateElement._flatpickr.setDate(firstDate);
```

Comment:
Just do this:

```suggestion
            endDateElement._flatpickr.setDate(startDateElement._flatpickr.selectedDates[0])
```

Does that make sense?

Reply:
Yes.  My confusion was thinking the startDate value was a string.